error with Envoy grpc

Put .evans.toml with following code under  https://github.com/envoyproxy/java-control-plane/tree/master/api/src/main/proto   

```
[default]
protoFile = ["envoy/api/v2/lds.proto"]
package = "envoy.api.v2"
service = "ListenerDiscoveryService"
```

with following stacktrace: 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x14fded9]

goroutine 20 [running]:
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.(*parseResult).addMessageDecls(0xc4203a06a0, 0xc4202a4690, 0xc420384ca0, 0xc420371740, 0x6, 0x8, 0xc4200f9c01)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:844 +0xbd9
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.(*parseResult).asMessageDescriptor(0xc4203a06a0, 0xc420384c60, 0x1, 0x0)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:812 +0x17e
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.(*parseResult).asFileDescriptor(0xc4203a06a0, 0xc420282940, 0x1f, 0xc42039c5a0, 0xc420324500, 0x17f1f00, 0xc420357380)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:522 +0xa04
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.createParseResult(0xc420282940, 0x1f, 0xc42039c5a0, 0xc4202f5000, 0x1000, 0x1000)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:493 +0xa6
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.parseProto(0xc420282940, 0x1f, 0x17f0020, 0xc4200be8f0, 0xc4200be8f0, 0xc4202437f0, 0x176d164)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:478 +0x1ee
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.parseProtoFiles.func1(0x17f34c0, 0xc4200be8f0, 0xc4201036d8, 0xc4202b86f0, 0xc4201036e8)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:288 +0xb5
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.parseProtoFiles(0xc4202b86c0, 0xc4200f9600, 0x8, 0x8, 0xc4202b86f0, 0x0, 0x28)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:289 +0x241
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.parseProtoFiles(0xc4202b86c0, 0xc4202997d0, 0x1, 0x1, 0xc4202b86f0, 0x20, 0x3265000001642340)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:293 +0x2cc
github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse.Parser.ParseFiles(0xc4202437e0, 0x2, 0x2, 0x0, 0x0, 0x0, 0xc4202997d0, 0x1, 0x1, 0x0, ...)
	/go/src/github.com/ktr0731/evans/vendor/github.com/jhump/protoreflect/desc/protoparse/parser.go:140 +0xb7
github.com/ktr0731/evans/adapter/internal/protoparser.ParseFile(0xc4202997d0, 0x1, 0x1, 0xc4202437e0, 0x2, 0x2, 0x15379ce, 0x1642340, 0x0, 0x1, ...)
	/go/src/github.com/ktr0731/evans/adapter/internal/protoparser/parser.go:28 +0x40e
github.com/ktr0731/evans/adapter/parser.ParseFile(0xc4202997d0, 0x1, 0x1, 0xc4202997c0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/ktr0731/evans/adapter/parser/parser.go:10 +0x81
github.com/ktr0731/evans/di.initEnv.func1()
	/go/src/github.com/ktr0731/evans/di/dependency.go:32 +0x10e
sync.(*Once).Do(0x1bc76f0, 0xc420055cd0)
	/usr/local/go/src/sync/once.go:44 +0xbe
github.com/ktr0731/evans/di.initEnv(0xc42024b900, 0x0, 0x0)
	/go/src/github.com/ktr0731/evans/di/dependency.go:24 +0x5f
github.com/ktr0731/evans/di.Env(0xc42024b900, 0xc420055d58, 0x1027b09, 0xc4200ac070, 0xc420055d78)
	/go/src/github.com/ktr0731/evans/di/dependency.go:59 +0x2b
github.com/ktr0731/evans/di.initPromptInputter.func1()
	/go/src/github.com/ktr0731/evans/di/dependency.go:149 +0x3c
sync.(*Once).Do(0x1bc7740, 0xc420055db0)
	/usr/local/go/src/sync/once.go:44 +0xbe
github.com/ktr0731/evans/di.initPromptInputter(0xc42024b900, 0x0, 0x0)
	/go/src/github.com/ktr0731/evans/di/dependency.go:147 +0x5f
github.com/ktr0731/evans/di.initDependencies.func1.2(0x0, 0x0)
	/go/src/github.com/ktr0731/evans/di/dependency.go:217 +0x2a
github.com/ktr0731/evans/di.(*initializer).init(0xc4202b8660, 0xc420055e90, 0x1050700)
	/go/src/github.com/ktr0731/evans/di/dependency.go:200 +0x8d
github.com/ktr0731/evans/di.initDependencies(0xc42024b900, 0x17f0020, 0xc4200be000, 0x0, 0xc420055f00)
	/go/src/github.com/ktr0731/evans/di/dependency.go:224 +0x7e
github.com/ktr0731/evans/di.NewREPLInteractorParams(0xc42024b900, 0x17f0020, 0xc4200be000, 0x0, 0x0, 0x0)
	/go/src/github.com/ktr0731/evans/di/interactor.go:24 +0x43
github.com/ktr0731/evans/adapter/controller.(*CLI).runAsREPL.func2(0xc420299720, 0xc4200c5c20, 0xc4202c0360, 0x17f6700, 0xc42024b980, 0xc42009e120)
	/go/src/github.com/ktr0731/evans/adapter/controller/cli.go:375 +0xe3
created by github.com/ktr0731/evans/adapter/controller.(*CLI).runAsREPL
	/go/src/github.com/ktr0731/evans/adapter/controller/cli.go:359 +0x185
```

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -92,7 +92,7 @@
     "dynamic",
     "internal"
   ]
-  revision = "dec7efff960aff9689f61d913a23286a260ac875"
+  revision = "86d31fcaca06045b722f08bae124c96548404121"
 
 [[projects]]
   name = "github.com/kelseyhightower/envconfig"
@@ -272,13 +272,13 @@
     "internal/timeseries",
     "trace"
   ]
-  revision = "a680a1efc54dd51c040b3b5ce4939ea3cf2ea0d1"
+  revision = "3673e40ba22529d22c3fd7c93e97b0ce50fa7bdd"
 
 [[projects]]
   branch = "master"
   name = "golang.org/x/sys"
   packages = ["unix"]
-  revision = "ac767d655b305d4e9612f5f6e33120b9176c4ad4"
+  revision = "bd9dbc187b6e1dacfdd2722a87e83093c2d7bd6e"
 
 [[projects]]
   name = "golang.org/x/text"
@@ -311,7 +311,7 @@
     "protobuf/ptype",
     "protobuf/source_context"
   ]
-  revision = "fedd2861243fd1a8152376292b921b394c7bef7e"
+  revision = "2a72893556e4d1f6c795a4c039314c9fa751eedb"
 
 [[projects]]
   name = "google.golang.org/grpc"

--- a/meta/meta.go
+++ b/meta/meta.go
@@ -7,5 +7,5 @@ import (
 const AppName = "evans"
 
 var (
-	Version = semver.MustParse("0.4.8")
+	Version = semver.MustParse("0.4.9")
 )

